specification/syntex/resource-manager/Microsoft.Syntex/preview/2022-09-15-preview/examples/DocumentProcessor_Create.json

```diff
@@ -0,0 +1,39 @@
+{
+  "parameters": {
+    "subscriptionId": "cd0d3cc8-cae2-4559-b882-cd707170799b",
+    "resourceGroupName": "mygroup",
+    "processorName": "myprocessor",
+    "api-version": "2022-09-15-preview",
+    "body": {
+      "location": "westus",
+      "tags": {
+        "additionalProp1": "string1",
+        "additionalProp2": "string2",
+        "additionalProp3": "string3"
+      },
+      "properties": {
+        "spoTenantUrl": "https://test123.sharepoint.com",
+        "spoTenantId": "e9bb744b-9558-4dc6-9e50-a3297e3332fa"
+      }
+    }
+  },
+  "responses": {
+    "200": {
```

Comment:
>200 [](http://example.com/codeflow?start=5&length=3)

Add 201 response example?

Reply:
It seems that returning 201 makes PUT an asynchronous operation, and we agreed at the ARM review board meeting earlier this year that this PR will not b asynchronous.

Reply:
No, returning 201 Created for PUT is just following standard HTTP behavior. Returning 202 makes PUT asynchronous. You may be thinking of returning 201 with nonterminal provisioning status or returning Azure-AsyncOperation headers with 201 which are only needed for async operations. Please review the RPC for further clarification.